.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+  - repo: local
+    hooks:
+      - id: changelog_verify
+        name: Verify if CHANGELOG.md has been updated
+        entry: python ./scripts/pre-commit/validate_changelog_update.py
```

Comment:
Nit: the program name could be `changelog_validator.py` or just `validate_changelog`. The id also could be `validate-changelog` or `check-changelog` since this is a check.